legal-api/src/legal_api/models/request_tracker.py

```diff
@@ -96,11 +96,16 @@ def find_by(  # pylint: disable=too-many-arguments
         filing_id: int = None,
         is_admin: bool = None,
         message_id: str = None,
+        business: any = None,
```

Comment:
can we remove `legal_entity_id` from the parameter list?

Reply:
Yes, I can do it.
But, `queue-service/entity-bn` used this func
https://github.com/kzdev420/lear/blob/a1b1c98c8ceb8d5516a5e3b385ab8b9d07817a91/queue_services/entity-bn/src/entity_bn/bn_processors/admin.py#L41-L47

If I updated the request_tracker model, should I update the queue-service too?

Reply:
don't worry about other services for now.  we have tickets for that work.